Source/WebCore/animation/AnimationTimelinesController.cpp

```diff
@@ -310,42 +311,46 @@ void AnimationTimelinesController::maybeClearCachedCurrentTime()
         m_cachedCurrentTime = std::nullopt;
 }
 
-void AnimationTimelinesController::registerNamedScrollTimeline(const AtomString& name, Element& source, ScrollAxis axis)
+static Element* originatingElement(Ref<AnimationTimeline> timeline)
```

Comment:
This should take in a `Ref<ScrollTimeline>&`.

Reply:
Fixed.